server/base/src/main/java/org/apache/accumulo/server/fs/VolumeUtil.java

```diff
@@ -166,7 +165,7 @@ public TabletFiles(String dir, List<LogEntry> logEntries,
   public static String switchRootTableVolume(ServerContext context, String location)
       throws IOException {
     String newLocation = switchVolume(location, FileType.TABLE,
-        ServerConstants.getVolumeReplacements(context.getConfiguration()));
+        ServerConstants.getVolumeReplacements(context.getConfiguration(), context.getHadoopConf()));
```

Comment:
Could only context be passed here?

Reply:
In some cases yes but we still use this in Initialize where we don't have a good way to create ServerContext yet.